classes/Domain/CallForProposal.php

```diff
@@ -2,7 +2,9 @@
 
 namespace OpenCFP\Domain;
 
-use DateTime;
+use DateTimeInterface;
```

Comment:
please reorder `use` statements in alphabetical order to avoid Travis triggering Code Sniffer `ordered_imports` error.